web/package.json

```diff
@@ -19,7 +22,14 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc",
+    "fix": "gts fix",
+    "prepare": "yarn run compile",
+    "pretest": "yarn run compile",
```

Comment:
Both prepare and pretest map to same command. Is it intended?

Reply:
Hmm they are both added by `npx gts init`. I just left it there because I thought it might be some google convention thing 🤔  haha